### DISCUSSION TITLE
Не понравилось, как ведут собеседования.

### DISCUSSION THREAD

**Discussion by manas-anarov** [ivelum/job#24] [General]

**Comment by stebunovd** [comment]
Добрый день! Мне очень жаль что у вас остались неприятные впечатления от собеседования. Подскажите пожалуйста, что мы могли бы улучшить?

**Reply by manas-anarov** [reply]
Я извиняюсь был очень зол. Плохо написал решил исправить. HR сказала задать вопросы, я задал, как проходит стандартный день программиста в компании, на что получаю, откуда я знаю )))

**Reply by oileb** [reply]
Аким, очень сожалею, что мы с вами не поняли друг друга. Мне показалось, что вы спрашиваете про рабочий график и наличие всевозможных треккеров времени, и я, судя по всему, неудачно пошутила, имея в виду, что рабочий график действительно свободный, как и заявлено в нашей вики. Но теперь я понимаю, что вы, наверное, имели в виду что-то другое и шутка была не к месту. Приношу свои извинения!

**Reply by manas-anarov** [reply]
А кстати, она сказала не откуда я знаю. А понятия не имею. ) Я аж побледнел
